### PR TITLE
change storage of ssh pem from configmap to secret for ssh plugin

### PR DESCRIPTION
**Pull request by sivanzcw**
#549

* change pem backend storage from comfigmap to secret

* secret name is jobname-jobid-ssh

```
mxnet-job-v2-e7f18111-1cec-11ea-b688-fa163ec79500-ssh
```

* Remained configmap of job using ssh plugin will be deleted by gc if job is deleted

### CODE REVIEW

File: pkg/apis/helpers/helpers.go [hunk 1]
```diff
@@ -145,6 +163,16 @@ func DeleteConfigmap(job *vcbatch.Job, kubeClients kubernetes.Interface, cmName
 	return nil
 }
 
+// DeleteSecret delete secret
+func DeleteSecret(job *vcbatch.Job, kubeClients kubernetes.Interface, secretName string) error {
+	err := kubeClients.CoreV1().ConfigMaps(job.Namespace).Delete(secretName, nil)
```
Comment (hzxuzhonghu) [inline pkg/apis/helpers/helpers.go]: why configmaps?

Comment (sivanzcw) [reply pkg/apis/helpers/helpers.go]: error coding, modified it, thanks

**Comment by k82cn** [comment]
LGTM overall, please help to fix CI error :)

**Pull request merged**